### PR TITLE
deleted test3

### PR DESCRIPTION
**Pull request by vempatikalyan**
deleted test3

### CODE REVIEW

File: test3.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-test
```
Comment (vempatikalyan) [inline test3.txt:1]: why?

Comment (vempatikalyan) [reply test3.txt:1]: required changes

File: test3.txt
Comment (vempatikalyan) [inline test3.txt:1]: why did you delete this file?

**Comment by nagarjuna-immadi** [comment]
test

**Review by nagarjuna-immadi — APPROVED** [review]
Approved
